src/renderer/api/endpoints/metrics.api.ts

```diff
@@ -78,10 +78,15 @@ export function normalizeMetrics(metrics: IMetrics, frames = 60): IMetrics {
       result.forEach(res => {
         if (!res.values || !res.values.length) return;
 
+        let now = moment().unix();
+
         while (res.values.length < frames) {
-          const timestamp = moment.unix(res.values[0][0]).subtract(1, "minute").unix();
+          const timestamp = moment.unix(now).subtract(1, "minute").unix();
 
-          res.values.unshift([timestamp, "0"]);
+          if (!res.values.find((value) => value[0] === timestamp)) {
```

Comment:
I wonder how the comparison between `moment`'s work? Does `unix()` return a number? 

```suggestion
          if (!res.values.find(([value]) => value === timestamp)) {
```

Won't this sometimes break if the metrics values are from "now - (5 minutes + 3 seconds)" to "now"?

Reply:
`unix()` returns a number. "5 minutes + 3 seconds" is a problem, we should probably round the initial value properly.